src/Infrastructure/Migrations/Version20200401192701.php

```diff
@@ -22,12 +22,13 @@ public function up(Schema $schema) : void
         // this up() migration is auto-generated, please modify it to your needs
         $this->abortIf($this->connection->getDatabasePlatform()->getName() !== 'mysql', 'Migration can only be executed safely on \'mysql\'.');
 
-        $this->addSql('ALTER TABLE `order` ADD updated_date DATETIME DEFAULT NULL COMMENT \'(DC2Type:datetime_immutable)\', CHANGE created_at created_date DATETIME NOT NULL COMMENT \'(DC2Type:datetime_immutable)\'');
-        $this->addSql('ALTER TABLE maker ADD created_date DATETIME NOT NULL COMMENT \'(DC2Type:datetime_immutable)\', ADD updated_date DATETIME DEFAULT NULL COMMENT \'(DC2Type:datetime_immutable)\'');
-        $this->addSql('ALTER TABLE requester ADD created_date DATETIME NOT NULL COMMENT \'(DC2Type:datetime_immutable)\', ADD updated_date DATETIME DEFAULT NULL COMMENT \'(DC2Type:datetime_immutable)\'');
-        $this->addSql('ALTER TABLE user ADD created_date DATETIME NOT NULL COMMENT \'(DC2Type:datetime_immutable)\', ADD updated_date DATETIME DEFAULT NULL COMMENT \'(DC2Type:datetime_immutable)\'');
-        $this->addSql('ALTER TABLE thing ADD created_date DATETIME NOT NULL COMMENT \'(DC2Type:datetime_immutable)\', ADD updated_date DATETIME DEFAULT NULL COMMENT \'(DC2Type:datetime_immutable)\'');
-        $this->addSql('ALTER TABLE commitment ADD updated_date DATETIME DEFAULT NULL COMMENT \'(DC2Type:datetime_immutable)\', CHANGE created_at created_date DATETIME NOT NULL COMMENT \'(DC2Type:datetime_immutable)\'');
+        $this->addSql('SET SQL_MODE=\'ALLOW_INVALID_DATES\';');
```

Comment:
Why?

Reply:
altering existing entries was going to insert Dates as. `0000-00-00 00:00:00` which is invalid. the migration could not complete the other tasks. that's why I hotfixed this setting for the migration. I have already checked the mysql settings on production and this value is not active anymore.